lib/utf8.pm

```diff
@@ -1,15 +1,19 @@
 package utf8;
 
-$utf8::hint_bits = 0x00800000;
+use strict;
+use warnings;
 
-our $VERSION = '1.23';
+our $hint_bits = 0x00800000;
```

Comment:
this is `HINT_UTF8` at minimum we probably want a test to confirm it's in sync with `perl.h`

Reply:
Right. This sounds like a separate case.